falcon/testing/helpers.py

```diff
@@ -107,17 +108,21 @@ class ASGIRequestEventEmitter:
             events. May be an empty string. If a byte string, it will
             be used as-is; otherwise it will be encoded as UTF-8
             (default b'').
-        disconnect_at (int): The Unix timestamp after which to begin
-            returning http.disconnect events (default now + 30s).
         chunk_size (int): The maximum number of bytes to include in
             a single http.request event (default 4096).
+        disconnect_at (int): The Unix timestamp after which to begin
```

Comment:
Looking at the code, `float` seems to be implicitly supported too. And maybe it is even desirable to support fractional seconds here?